tests/benchmarks/variable_length_filter.yml

```diff
@@ -0,0 +1,23 @@
+name: "VARIABLE-LENGTH-FILTER"
+remote:
+  - setup: redisgraph-r5
+  - type: oss-standalone
+dbconfig:
+  - dataset: "datasets/highly_connected.rdb"
+clientconfig:
+  - tool: redisgraph-benchmark-go
+  - parameters:
+    - graph: "g"
+    - rps: 0
```

Comment:
`0` stands for unlimited?

Reply:
yes :+1: . Not specifying rps also has the same effect